src/app/item-popup/PlugTooltip.tsx

```diff
@@ -20,7 +19,10 @@ import Objective from '../progress/Objective';
 import './ItemSockets.scss';
 
 // TODO: Connect this to redux
-export default function PlugTooltip({
+/**
+ * This renders the tootip content for a DimPlug.
```

Comment:
I sorta figured, from the name

Reply:
Haha yeah I had that before I renamed it. Will remove.w